src/main/java/nextstep/courses/domain/session/Session.java

```diff
@@ -0,0 +1,55 @@
+package nextstep.courses.domain.session;
+
+import nextstep.courses.domain.strategy.PaymentStrategy;
+import nextstep.payments.domain.Payment;
+
+import java.time.LocalDate;
+import java.util.Objects;
+
+public class Session {
+
+    private final SessionImage sessionImage;
+    private final SessionState sessionState;
+    private final PaymentStrategy sessionStrategy;
+    private final SessionDate sessionDate;
+
+    public Session(int size, String imageType, int width, int height, SessionState sessionState, PaymentStrategy sessionStrategy, LocalDate startDate, LocalDate endDate) {
+        this(new SessionImage(size, imageType, width, height), sessionState, sessionStrategy, new SessionDate(startDate, endDate));
+    }
+
+    public Session(SessionImage sessionImage, SessionState sessionState, PaymentStrategy sessionStrategy, SessionDate sessionDate) {
+        this.sessionImage = sessionImage;
+        this.sessionState = sessionState;
+        this.sessionStrategy = sessionStrategy;
+        this.sessionDate = sessionDate;
+    }
+
+    public boolean applyForCourse(Payment payment, LocalDate date) {
```

Comment:
```suggestion
    public boolean canApplyForCourse(Payment payment, LocalDate date) {
```

위처럼 가능 여부를 리턴하기에 `can` 을 붙여 의미전달하는건 어떨까요?
`applyForCourse` 라고한다면 apply 되는것 같아요!

Reply:
can 을 추가해서 가능 여부의 의미를 전달하는 방법 정말 좋은것 같네요!!!